packages/fluentui/e2e/tests/dialogInDialog.spec.ts

```diff
@@ -0,0 +1,95 @@
+// https://github.com/microsoft/fluent-ui-react/issues/1674
+describe('Dialog in Dialog', () => {
+  const selectors = {
+    outerClose: 'outer-close',
+    outerHeader: 'outer-header',
+    outerTrigger: 'outer-trigger',
+    innerClose: 'inner-close',
+    innerHeader: 'inner-header',
+    innerTrigger: 'inner-trigger',
+    overlayPoint: 'overlay-point',
+  };
+
+  const outerClose = `#${selectors.outerClose}`;
+  const outerHeader = `#${selectors.outerHeader}`;
+  const outerTrigger = `#${selectors.outerTrigger}`;
+  const innerClose = `#${selectors.innerClose}`;
+  const innerHeader = `#${selectors.innerHeader}`;
+  const innerTrigger = `#${selectors.innerTrigger}`;
+  const overlayPoint = `#${selectors.overlayPoint}`;
```

Comment:
off-topic: I don't know will be an exact approach with selectors, but we will need to discuss it in offline.